_posts/2021-05-07-infinispan-12.1.2.Final.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog
+title: Infinispan 12.1.2.Final
+permalink: /blog/:year/:month/:day/infinispan-12-1-2-final
+date: '2021-05-07T12:00:00.000-00:00'
+author: karesti
+tags: [ "release" ]
+---
+
+= Infinispan 12.1.2.Final
+
+Dear Infinispan community,
+
+It's been a month since our first 12.1.0.Final release and we already released two additional minor released with
+some bug fixes and upgrades. The latest version is 12.1.2.Final.
+
+Highlights if these two minor releases include:
+
+* Grafana Dashboards for the Operator https://issues.redhat.com/browse/ISPN-12702[ISPN-12702]
+* Performance regression fix https://issues.redhat.com/browse/ISPN-13017[ISPN-13017]
+* The Infinispan Web Console displays now https://issues.redhat.com/browse/ISPN-12458[Index and Query improved statistics].
```

Comment:
I think this needs rewording.

"Index and Query statistics improved in the Infinispan Web Console"?

Reply:
ok